Add german translation (de)

iBroLab opened the pull request:
Hello tonymanou,

here are the german translation for the next update of Screen Filter.
I hope you like it.

Commit:
German translation

Hello tonymanou,

here are the german translation for the next update of Screen Filter.

tonymanou commented:
Hello iBroLab, great, thank you for your help!
You can put your name in the `about_translator` field (line 9) if you want.

There is a strange character `´` at line 67, but you kept the standard apostrophe `'` at line 70. Is it on purpose?
Could you update your pull request?
(Note: If you are not confident with GitHub, go to [the branch that was created when you made the pull request](https://github.com/iBroLab/ScreenFilter/blob/patch-1/res/values-de/strings.xml), edit what you need and commit, the changes will be added to this pull request.)

Commit:
Update german translation

To your request I replaced the ` ´  `  character in line 67 with the standard apostrophe ` ' ` (that was'nt my purpose to use character ` ´ `  ).
I also add my name in the ` about_translator ` fied and changed some strings (there was some grammatical errors in my german translation).

ghost commented:
To your request I replaced the `´`  character in line 67 with the standard apostrophe `'` (that was'nt my purpose to use character `´`  ).
I also add my name in the `about_translator` fied and changed some strings (there was some grammatical errors in my german translation).

tonymanou commented:
Thank you!
